src/store/dashboard/awsDashboard/awsDashboardWidgets.ts

```diff
@@ -66,6 +67,7 @@ export const costSummaryWidget: AwsDashboardWidget = {
     computedReportItemValue: ComputedReportItemValueType.total,
     dailyTitleKey: messages.awsDailyCostTrendTitle,
     titleKey: messages.awsCostTrendTitle,
+    chartName: 'awsCostTrend',
```

Comment:
Would you mind prefixing the chart names with "aws"; for example, "awsCostChart"